src/components/BottomNav.js

```diff
@@ -1,23 +1,55 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 
-function BottomNav() {
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import FormatListBulletedIcon from '@material-ui/icons/FormatListBulleted';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    minWidth: 375,
+    maxWidth: 3000,
+    height: 30,
+  },
+  tab: {},
```

Comment:
Looks great!! Maybe we could delete this `tab` class if we don't need it, and perhaps the maxWidth for the navbar could be smaller.  Next week, it would be great if we could put the navbar right under the list - but I know that's outside the scope of this issue.